Makefile

```diff
@@ -41,6 +43,7 @@ define package
 		--build-arg ITERATION=$(iteration) \
 		--build-arg PACKAGE_VERSION=$(version) \
 		--build-arg PACKAGE_TYPE=$(2) \
+		--build-arg DIST=${dist}
```

Comment:
Is a slash missing?

Reply:
> Is a slash missing?

@EnableAsync good catch and updated. Thanks.